### PR TITLE
SPCTR-260 Feature: Analytics support in Spectra

### PR DESCRIPTION
**Pull request by sushmak02**
### Description
Logic  - Save the version number to know when the user updated the page so that we can remove all deprecated parts and backwards CSS

Which data can be collected - 
- Admin Settings
- Blocks Used on pages Count
- Version number

### Types of changes

Feature

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by RajkiranBagal — COMMENTED** [review]
Did we miss deleting this option `spectra_settings_data` after sometime & send updated data?

**Comment by sushmak02** [comment]
> @sushmak02 Did we miss deleting this option `spectra_settings_data` after sometime & send updated data?

Done

File: admin-core/inc/admin-menu.php [hunk 1]
```diff
@@ -104,6 +111,15 @@ public function settings_admin_scripts() {
```
Comment (AbhijitNage123) [inline admin-core/inc/admin-menu.php:111]: Update a description for settings_admin_scripts() to `Initialize after Spectra get loaded.`

**Review by RajkiranBagal — APPROVED** [review]

File: admin-core/inc/admin-menu.php (hunk 1 above)
Comment (sushmak02) [reply admin-core/inc/admin-menu.php:112]: done

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**